admin/etc/adminhtml/system.xml

```diff
@@ -36,6 +36,11 @@
                             <source_model>Pronko\LiqPayAdmin\Source\ConnectionType</source_model>
                             <config_path>payment/pronko_liqpay/connection_type</config_path>
                         </field>
+                        <field id="payment_action" translate="label" type="select" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="0" canRestore="1">
+                            <label>Payment Action</label>
+                            <source_model>Magento\AuthorizenetAcceptjs\Model\Adminhtml\Source\PaymentAction</source_model>
```

Comment:
Payment Action should be created in the Pronko_LiqPayAdmin module

Reply:
Fixed